jenkins/ci.suse.de/pipelines/openstack-ardana-bootstrap.Jenkinsfile

```diff
@@ -87,6 +87,23 @@ pipeline {
       }
     }
 
+    stage('ansible linting') {
+      steps {
+        sh '''
+          set -x
```

Comment:
The general trend is towards having as much as possible implemented by ansible playbooks rather than bash scripts. This has several advantages: 
- ansible already has support for pip/virtualenv
- almost all of the other pipeline stages are implemented by ansible playbooks, which means an ansible playbook can be better integrated into the overall development workflow (i.e. running the same scripts called by the Jenkins pipeline from your local workstation)
- consistent logging
- many other

The CI ansible playbooks are located in `scripts/jenkins/ardana/ansible`.